start 1688741765.8305383
gen 1688741765.8435369
end 1688741765.8519373
Traceback (most recent call last):
  File "/home/yasmin/constraintflow/backward/experiments.py", line 33, in <module>
    genAST(sys.argv[1],int(a[0]), int(a[1]), int(a[2]))
  File "/home/yasmin/constraintflow/backward/experiments.py", line 30, in genAST
    v.visit(ast)
  File "/home/yasmin/constraintflow/backward/astVisitor.py", line 91, in visit
    return self.visitProg(node)
  File "/home/yasmin/constraintflow/backward/verify.py", line 286, in visitProg
    self.visit(node.stmt)
  File "/home/yasmin/constraintflow/backward/astVisitor.py", line 87, in visit
    return self.visitSeq(node)
  File "/home/yasmin/constraintflow/backward/verify.py", line 282, in visitSeq
    self.visit(node.stmt2)
  File "/home/yasmin/constraintflow/backward/astVisitor.py", line 89, in visit
    return self.visitFlow(node)
  File "/home/yasmin/constraintflow/backward/verify.py", line 208, in visitFlow
    self.applyTrans([], vallist, s, curr_prime, computation)
  File "/home/yasmin/constraintflow/backward/verify.py", line 255, in applyTrans
    self.applyTrans(leftC + [condz3], vallist.left, s, curr_prime, computation)
  File "/home/yasmin/constraintflow/backward/verify.py", line 251, in applyTrans
    raise Exception("Transformer"+ " " + " not true")
Exception: Transformer  not true
real 0.20
user 0.17
sys 0.04
